.github/CHANGELOG.md

```diff
@@ -71,6 +73,10 @@ Aroosa Ijaz
 
 ### Breaking changes
 
+* The following CV observables were renamed to comply with the new Operation/Observable
```

Comment:
These should go in v0.5.0-dev, no?

Reply:
Also an artifact of the merge of observable_name_refactor